doc/changes/latest.inc

```diff
@@ -62,6 +62,8 @@ Enhancements
 
 - In :class:`mne.Report`, limit the width of automatically generated figures to a maximum of 850 pixels (450 pixels for :class:`mne.SourceEstimate` plots), and the resolution to 100 DPI to reduce file size, memory consumption, and – in some cases like :meth:`mne.Report.add_stc` – processing time (:gh:`10126`, :gh:`10129`, :gh:`10135` by `Richard Höchenberger`_)
 
+  - Annotations from a :class:`~mne.io.Raw` object are now preserved by the :class:`~mne.Epochs` constructor (:gh:`9969` by `Adam Li`_)
```

Comment:
```suggestion
 - Annotations from a :class:`~mne.io.Raw` object are now preserved by the :class:`~mne.Epochs` constructor (:gh:`9969` by `Adam Li`_)
```

Errant space made this indented

https://39914-1301584-gh.circle-artifacts.com/0/dev/whats_new.html